src/components/AnimatedSignal/README.md

```diff
@@ -0,0 +1,71 @@
+# Animated Signal
+
+`Animated Signal` Component gives animation to user motion, color, and visual cues to capture the user's attention and deliver important information in an engaging and intuitive manner.
+
+#### 🎬 Preview
+
+|          Animated Signal           |
+| :--------------------------------: |
+| ![alt_tag](/assets/WifiSignal.gif) |
+
+### Default usage
+
+---
+
+```jsx
+import React from "react";
+import { AnimatedSignal } from "react-native-animation-catalog";
+
+const AnimatedSignal = () => <AnimatedSignal />;
```

Comment:
@het-a-simform Don't we need any props, nothing is required in this component ?

Reply:
This code gives basic usage of our component in custom usage we pass total 3 props.